lib/pip.js

```diff
@@ -122,6 +123,14 @@ function installRequirements(
     cmd = pipCmd[0];
     cmdOptions = pipCmd.slice(1);
   }
+
+  // If enabled slimming, strip out the caches, tests and dist-infos
+  if (options.slim === true || options.slim === 'true') {
+    const preparedPath = dockerPathForWin(options, targetRequirementsFolder);
+    const slimCmd = getSlimPackageCommands(options, preparedPath);
```

Comment:
Now the preparing of the strip command happens inside of this method - in case when the environment does not meet expectations (`win32` systems) returns an empty array. 

To be honest, I am slightly worried about this - theoretically running these commands inside of CYGWIN or Git Bash (or other bash emulators) should be just fine - but I couldn't find a way yet to check for this... would appreciate any advice! :)